include/ttmlir/Dialect/TTIR/IR/TTIROps.td

```diff
@@ -954,7 +960,8 @@ def TTIR_MaxPool2dOp : TTIR_DPSOp<"max_pool2d"> {
                          SI32Attr:$padding_left,
                          SI32Attr:$padding_right,
                          SI32Attr:$padding_top,
-                         SI32Attr:$padding_bottom);
+                         SI32Attr:$padding_bottom,
+                         DefaultValuedAttr<I64Attr, "0">:$channel_last);
```

Comment:
Is there a particular reason for choosing `I64Attr` instead of `BoolAttr` here?

Reply:
Well kinda yes and no. It started as boolattr but I couldn't get op to build. In the process of trying to figure out what is the issue I changed it to I64Attr and forgot to revert it once I figured out what is the issue. I will changed it.